### DISCUSSION TITLE
Toast在暗色模式下的背景色是否要改成不透明？

### DISCUSSION THREAD

**Discussion by zdpcdt** [DouyinFE/semi-design#2473] [Q&A] [answered]
`Toast`和`Notification`是相似的控件，但是当两者的`theme`都是`light`的时候，在暗色模式下，`Notification`的背景色不是透明，而`Toast`的背景色是透明的。

Notification:
![image](https://github.com/user-attachments/assets/6fdad614-0356-42c1-bc85-965ad97c2838)

Toast:
![image](https://github.com/user-attachments/assets/f98b48fa-4093-4464-b9ef-363440876a6f)

两者的设计是否应该统一呢？我个人倾向于不透明的设计，这样比较符合直觉。

**Comment by SudoUserReal** [comment, marked as answer]
我也倾向于不透明的设计，这样比较符合直觉。

**Comment by SudoUserReal** [comment]
我们改一下
